deployment/README.md

```diff
@@ -94,3 +94,66 @@ bash-4.4# ./scripts/infra apply
 ```
 
 This will attempt to apply the plan assembled in the previous step using Amazon's APIs.
+
+## Logs and Debugging
+
```

Comment:
Consider mentioning the RequestLog table in the OAR database, accessible via the Django Admin that is used to count and report on requests made with an API token.

Reply:
Added